pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -82,39 +88,40 @@ func (c *Reconciler) createOrUpdateAffinityAssistantsAndPVCs(ctx context.Context
 	// affinity assistant so that the OwnerReference of the PVCs are the pipelineruns, which is used to achieve PVC auto deletion at PipelineRun deletion time
 	if (aaBehavior == aa.AffinityAssistantPerWorkspace || aaBehavior == aa.AffinityAssistantDisabled) && pr.HasVolumeClaimTemplate() {
 		if err := c.pvcHandler.CreatePVCsForWorkspaces(ctx, pr.Spec.Workspaces, *kmeta.NewControllerRef(pr), pr.Namespace); err != nil {
-			return fmt.Errorf("failed to create PVC for PipelineRun %s: %w", pr.Name, err)
+			return fmt.Errorf("%w: %v", ErrPvcCreationFailed, err) //nolint:errorlint
```

Comment:
nit: you can use %s with `err.Error` rather than ignoring a linter

Reply:
Seems like change to %s still give me `errorlint` when running `golangci-lint`:

``` go			
return fmt.Errorf("%w: %s", ErrPvcCreationFailed, err.Error())
```

```
golangci-lint run --modules-download-mode=vendor --max-issues-per-linter=0 --max-same-issues=0 --deadline 5m

pkg/reconciler/pipelinerun/affinity_assistant.go:91:54: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors (errorlint)
                        return fmt.Errorf("%w: %s", ErrPvcCreationFailed, err.Error())
```